src/main/configSchema.ts

```diff
@@ -1,15 +1,15 @@
 export type ConfigurationSchema = {
     storagePath: string,
-    bufferStoragePath?: string,
-    retailLogPath?: string,
-    classicLogPath?: string,
+    bufferStoragePath: string,
+    retailLogPath: string,
+    classicLogPath: string,
```

Comment:
Just to confirm my understanding here. Even if user doesn't set classicLogPath, it will end up in the config as an empty string, so this is fine?

Reply:
Correct. Empty strings should be considered an abxesnt value anyway for a path value.